pkg/xds/server/callbacks/dataplane_status_tracker.go

```diff
@@ -148,20 +174,40 @@ func (c *dataplaneStatusTracker) OnStreamRequest(streamID int64, req util_xds.Di
 		}
 	}
 
-	// update Dataplane status
 	subscription := state.subscription
+	log := statusTrackerLog.WithValues(
+		"proxyName", state.dataplaneId.Name,
+		"mesh", state.dataplaneId.Mesh,
+		"streamID", streamID,
+		"type", shortEnvoyType(req.GetTypeUrl()),
+		"version", req.VersionInfo(),
```

Comment:
You qualify the dpVersion may be worth qualifying this version (reqVersion? XDSVersion? I'm actually unsure what it is :)

Reply:
changed to `resourceVersion` (similar to `resourceCount`). Is it better?